src/core_atmosphere/physics/mpas_atmphys_driver_sfclayer.F

```diff
@@ -1519,7 +1527,7 @@ subroutine driver_sfclayer(itimestep,configs,mesh,diag_physics,sfc_input,its,ite
                    rho3d    = rho_p      , dz8w      = dz_p     , cp       = cp          , &
                    g        = gravity    , rovcp     = rcp      , R        = R_d         , &
                    xlv      = xlv        , chs       = chs_p    , chs2     = chs2_p      , &
-                   cqs2     = cqs2_p     , cpm       = cpm_p    , znt      = znt_p       , &
+                   cqs = cqs_p,cqs2     = cqs2_p     , cpm       = cpm_p    , znt      = znt_p       , &
```

Comment:
Laura will despise this messing up her formatting.

Reply:
@joeolson42 you are right, Joe. I fixed it now. Thank you for bringing this to my attention.